### ISSUE TITLE
What is this?

### ISSUE THREAD

**Issue by TristanJamesBall** [informix/compose-kernel-lab#2]
Sorry, not sure where else to ask, but I'm very curious is to what this was for, or from? Was there some kind of low level informix training course that needed this kind of tooling?
If so, is it still available? 
I haven't been able to find anything obviously

**Comment by brian-hughes** [comment]
Yes, this was part of an Informix User Group training some 3 years ago.  If I recall it was a live training session and I don't have the materials for this to make this project make any sense.  I'll check with the team, but we will probably remove this and look at better demo's training for containers and cloud solutions.
